examples/Phonon_Caustics/Caustic.mac

```diff
@@ -0,0 +1,13 @@
+# This Macro is to Generate Phonons Caustics patterns
+/run/numberOfThreads 1
+/run/initialize
+/run/printProgress 100000
+/process/inactivate phononScattering
+/process/inactivate phononDownconversion
+/g4cmp/phononBounces 3
+#/tracking/verbose 2
+/gps/energy 0.03 eV
+/gps/number 1
+/gps/ang/type iso
+/gps/pos/centre 0.00 0.0 0.0 cm
```

Comment:
I would suggest moving these lines up to the beginning of your macro, before processes and worker threads get instantiated.  That way all of the final "after /run/init" stuff is at the end of the macro, right before the run starts.

Reply:
I changed the macro to this:
# This Macro is to Generate Phonons Caustics patterns
/gps/energy 0.03 eV
/gps/number 1
/gps/ang/type iso
/gps/pos/centre 0.0 0.0 0.0 cm
/run/numberOfThreads 1
/run/initialize
/g4cmp/phononBounces 100
/process/inactivate phononScattering
/process/inactivate phononDownconversion
/run/beamOn 40000
 
This is the only way that the program works

Reply:
That's odd!  In the CDMS simulations, the `/g4cmp/...` commands can be anywhere; we instantiate {{G4CMPConfigManager}} in our main(), just like you do (I thought! Let me check...)  Yes, you have that at [line 42](https://github.com/kelseymh/G4CMP/blob/82b5254362920699c03675eab17c9f5c42609e2d/examples/Phonon_Caustics/g4cmpPhononCaustics.cc#L42).  Is it really true that the `phononBounces` command doesn't work before `/run/initialize`?

Reply:
You are right. For this example, this command is not necessary, because the phonons are detected at the first hit. I have eliminated this command.